### ISSUE TITLE
Paginator page-size select is skipped by VoiceOver and JAWS

### ISSUE THREAD

**Issue by jelbourn** [angular/components#11139]
### Summary
Voiceover skips items per page list box

### Reproduction on material.angular.io
1. Linear navigate to ' Components ' menu item and click 'Enter' to open components page.                                                                                                                                                                                                                                          
2. Linear navigate forward and set focus on expand collapse left hand menu item ' Navigation'                                                                                                                                                                                                                                  
3. Click 'Enter' key to expand OR collapse this menu item                                                                                                                                                                                                                                                                                       
4. Linear navigate into 'Table' & click ENTER keystroke.                                                                                                                                                                                                                                                                                   
5. Navigate into tabs "Examples" and Click ENTER                                                                                                                                                                                                                                                                                 
6. Navigate to section ' tables with pagination' and navigate to 'items per page list box'                                                                                           Observe VO skips the items per page list box. in Jaws , user has to perform click OR escape operation twice respectively to open the list box and close the list box

**Comment by jelbourn** [comment]
I believe this should be fixed with the 2020 updates to mat-select.
